python/ql/src/semmle/python/web/twisted/Response.qll

```diff
@@ -8,29 +7,26 @@ import Request
 
 class TwistedResponse extends TaintSink {
     TwistedResponse() {
-        exists(PyFunctionObject func, string name |
+        exists(PythonFunctionValue func, string name, Return ret |
             isKnownRequestHandlerMethodName(name) and
             name = func.getName() and
             func = getTwistedRequestHandlerMethod(name) and
-            this = func.getAReturnedNode()
+            func.getScope() = ret.getScope() and
+            ret.getValue().getAFlowNode() = this
```

Comment:
Would it make sense to extend the `Value` API to also have a `getAReturnedNode` method?

Reply:
Yes. I think that would be nice. I'll do that tomorrow :+1:

Reply:
Done!